openmetadata-service/src/main/java/org/openmetadata/service/security/auth/BasicAuthenticator.java

```diff
@@ -469,6 +469,7 @@ public JwtResponse loginUser(LoginRequest loginRequest) throws IOException, Temp
     checkIfLoginBlocked(email);
     User storedUser = lookUserInProvider(email, loginRequest.getPassword());
     validatePassword(email, loginRequest.getPassword(), storedUser);
+    Entity.updateUserLastLoginTime(storedUser, System.currentTimeMillis());
```

Comment:
any reason to create this method in Entity ? instead of adding it to UserRepostiory, we can always refer Entity.getUserRepository()

Reply:
updated @harshach